cwl_wes/custom_config.py

```diff
@@ -3,22 +3,107 @@
 from typing import Dict, List, Optional
 from foca.models.config import FOCABaseConfig
 
+
 class StorageConfig(FOCABaseConfig):
+    """Model for task run and storage configuration.
+
+    Args:
+        tmp_dir: Temporary run directory path
+        permanent_dir: Permanent working directory path
+        remote_storage_url: Remote file storage FTP endpoint
+
+    Attributes:
+        tmp_dir: Temporary run directory path
+        permanent_dir: Permanent working directory path
+        remote_storage_url: Remote file storage FTP endpoint
+
+    Raises:
+        pydantic.ValidationError: The class was instantianted with an illegal
+            data type.
+
+    Example:
+        >>> StorageConfig(
+        ...     tmp_dir='/data/tmp',
+        ...     permanent_dir='/data/output',
+        ...     remote_storage_url='ftp://ftp.private/upload'
+        ... )
+        StorageConfig(tmp_dir='/data/tmp', permanent_dir='/data/output', remote_storage_url='ftp://ftp.private/upload')
+    """
     permanent_dir: str = '/data/output'
     tmp_dir: str = '/data/tmp'
     remote_storage_url: str = 'ftp://ftp-private.ebi.ac.uk/upload/foivos'
 
 
 class MonitorConfig(FOCABaseConfig):
+    """Model for monitoring celery.
+
+    Args:
+        timeout: Celery task timeout.
+
+    Attributes:
+        timeout: Celery task timeout.
+
+    Raises:
+        pydantic.ValidationError: The class was instantianted with an illegal
+            data type.
+
+    Example:
+        >>> MonitorConfig(
+        ...     timeout=15,
+        ... )
+        MonitorConfig(timeout=15)
+    """
     timeout: float = 0.1
```

Comment:
I don't really see the need (nor the semantic connection or necessity) to have a separate config model here. I would just put this in the base `CeleryConfig`.